### ISSUE TITLE
WARNING: Invalid cookie header

### ISSUE THREAD

**Issue by cinterloper** [SUSE/salt-netapi-client#163]
Is it normal to receive log messages like this?

```
Jul 01, 2016 10:32:55 PM org.apache.http.client.protocol.ResponseProcessCookies processCookies
WARNING: Invalid cookie header: "Set-Cookie: session_id=_MASKED_INFO_; expires=Sat, 02 Jul 2016 15:32:55 GMT; Path=/". Invalid 'expires' attribute: Sat, 02 Jul 2016 15:32:55 GMT
salt auth token: _MASKED_INFO_ Until: Sat Jul 02 10:32:55 MST 2016 Perms : [.*, @runner, @wheel, @jobs]
Jul 01, 2016 10:32:55 PM org.apache.http.client.protocol.ResponseProcessCookies processCookies
WARNING: Invalid cookie header: "Set-Cookie: session_id=_MASKED_INFO_; expires=Sat, 02 Jul 2016 15:32:55 GMT; Path=/". Invalid 'expires' attribute: Sat, 02 Jul 2016 15:32:55 GMT
```

**Comment by renner** [comment]
If haven't seen those warnings yet, what exactly would I do to reproduce this?

**Comment by cinterloper** [comment]
This occurs for me when i execute this piece of code
`https://github.com/cinterloper/vertx-salt/blob/master/src/main/resources/saltConnection.groovy`

thats an example that should connect to salt-api on localhost:8000 plaintext. 

the salt master i have configured with the following stanza:
```
rest_cherrypy:
  port: 8000
  debug: True
  disable_ssl: True
  expire_responses: False
```

to build that project (vertx-salt)
./gradlew clean shadoJar
to run 
 java -jar build/libs/vertx-salt-3.3.0-5-fat.jar

**Comment by cinterloper** [comment]
fwiw, this is the full salt-master config
```
rest_cherrypy:
  # can be any port
  port: 8000
  debug: True
  disable_ssl: True
  #ssl_crt: /etc/pki/tls/certs/localhost.crt
  #ssl_key: /etc/pki/tls/certs/localhost.key
  expire_responses: False
  
file_recv: True

ext_pillar:
  - file_tree:
      root_dir: /srv/pillar/tree
      follow_dir_links: False
      raw_data: False

file_roots:
  base:
    - /srv/salt

client_acl:
  user:
    - test.ping

external_auth:
  pam:
    user:
      - .*
      - '@runner'
      - '@wheel'
      - '@jobs'

log_level: debug

include: cloud.providers.d/*
include: cloud.profiles.d/*
```

**Comment by renner** [comment]
According to @adohren this should be fixed with 3bd1213f1502b6d797a52647676b38cb4b278547.
